packages/muelu/src/Smoothers/MueLu_BelosSmoother_decl.hpp

```diff
@@ -48,28 +48,28 @@
 
 #include <Teuchos_ParameterList.hpp>
 
-#include <Xpetra_BlockedCrsMatrix_fwd.hpp>
-#include <Xpetra_CrsMatrixWrap.hpp>
 #include <Xpetra_Matrix_fwd.hpp>
-#include <Xpetra_Matrix.hpp>
 #include <Xpetra_MultiVectorFactory_fwd.hpp>
-#ifdef HAVE_XPETRA_TPETRA
-#include <Xpetra_TpetraCrsMatrix.hpp>
-#endif
 
 #include "MueLu_ConfigDefs.hpp"
-#include "MueLu_BelosSmoother_fwd.hpp"
 
 #if defined(HAVE_MUELU_BELOS)
 
+#include "MueLu_BelosSmoother_fwd.hpp"
+
+#ifdef HAVE_XPETRA_TPETRA
+#include "BelosTpetraAdapter.hpp"
+#include <Tpetra_Operator.hpp>
```

Comment:
Use double quotes here too.  In general, only system headers get angle brackets.